integration-tests/src/test/java/org/apache/fineract/integrationtests/FixedDepositTest.java

```diff
@@ -2598,6 +2610,158 @@ public void testFixedDepositAccountWithRolloverPrincipal() {
         FixedDepositAccountStatusChecker.verifyFixedDepositIsActive(fixedDepositAccountStatusHashMap);
     }
 
+    @Test
+    public void testCloseFixedDepositForCLOSURE_TYPE_REINVEST_withConfigurationMaturityInstruction() {
```

Comment:
Please make sure any new test and internal API calls are using the fineract-client only! We should not write any tests which still using any of the deprecated helper methods or anything...unless it is a must!